packages/cli/README.md

```diff
@@ -34,6 +34,17 @@ glaze COMMAND
 - [`glaze model:inspect NAME`](#glaze-modelinspect-name)
 - [`glaze model:list`](#glaze-modellist)
 - [`glaze model:publish NAME [OUTPUT]`](#glaze-modelpublish-name-output)
+- [`glaze stream:pin:add STREAMID`](#glaze-streampinadd-streamid)
+- [`glaze stream:pin:rm STREAMID`](#glaze-streampinrm-streamid)
+- [`glaze stream:pin:ls STREAMID`](#glaze-streampinls-streamid)
+- [`glaze stream:schema:create CONTENT`](#glaze-streamschemacreate-content)
+- [`glaze stream:schema:update STREAMID CONTENT`](#glaze-streamschemaupdate-streamid-content)
```

Comment:
`schema:*` rather than `stream:schema:*`?

Reply:
Why do we even have these `schema` commands? Do they add much value over just creating tiles?

Reply:
@oed I don't think they add much value over the tiles beyond being slightly different in their implementation. I kept them here because they were easily replicated from the `stream` commands & they were in the Ceramic CLI. Can certainly remove them.